tower-rate-limit/Cargo.toml

```diff
@@ -7,7 +7,8 @@ publish = false
 [dependencies]
 futures = "0.1"
 tower-service = { version = "0.1", path = "../tower-service" }
-tokio-timer = "0.1"
+tokio-timer = "0.2"
```

Comment:
I've been switching to including the latest patch. We should probably be respecting "min versions" in our deps.

I wouldn't eagerly update any other dependency versions, just on a forward basis.

So this would be `tokio-timer = "0.2.6"`